src/Mod/Part/App/FeatureOffset.h

```diff
@@ -42,6 +42,7 @@ class PartExport Offset : public Part::Feature
     App::PropertyFloat Value;
     App::PropertyEnumeration Mode;
     App::PropertyEnumeration Join;
+    App::PropertyEnumeration Algo;
```

Comment:
Why was Algo property added to general (3d) offset class, and not to its 2d subclass? 3d offset, if ever, will likely have different enum values...